### ISSUE TITLE
Newer versions on Node need bcrypt removed from this module in order to work

### ISSUE THREAD

**Issue by thesteady** [crushlovely/utensils#1]
Hi @createbang ,

The `bcrypt` dependency doesn't appear to be used, and isn't compatible with Node 0.12. It should probably be removed. 

Thanks!

**Comment by heymichaelp** [comment]
Yep, @thesteady, I think you're right.  I'll submit a PR for this.
